src/App.js

```diff
@@ -1,13 +1,78 @@
 import React from 'react';
 
+import TodoForm from "./components/TodoComponents/TodoForm";
+import TodoList from "./components/TodoComponents/TodoList";
+import "./components/TodoComponents/Todo.css";
+
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+    this.state = {
+        todos: [],
+        // inputText: '',
+        // id: '',
+        // completed: ''
+    }
+}
+
+addTask = e => {
+  e.preventDefault();
+
+  const newTask = {
+      task: this.state.inputText,
+      id: Date.now(),
+      completed: false
+  };
+
+  this.setState({
```

Comment:
input, id, and completed are unneeded because you commented their state out and so you're then adding them back when you leave it in set state and this should fix your error